cypress/integration/seededFlows/moderationFlows/adjustPostTags.spec.js

```diff
@@ -117,7 +120,13 @@ describe('Adjust post tags', () => {
 
       cy.findByRole('button', { name: 'Moderation' }).click();
       cy.getIframeBody('#mod-container').within(() => {
-        cy.findByRole('button', { name: 'Open adjust tags section' }).click();
+        // Click listeners are attached async so we use pipe() to retry click until condition met
+        cy.findByRole('button', {
+          name: 'Open adjust tags section',
+        })
+          .pipe(click)
+          .should('have.attr', 'aria-expanded', 'true');
```

Comment:
The click event will be retried until the button's state updates to 'expanded'. It's quite nice that this pattern allows us to lean on testing good semantics/accessibility 😄  

(NB: the click event will only continue retrying according to the default timeout for the `should` condition)